### PR TITLE
Added symfony 5 support

### PR DESCRIPTION
**Pull request by chq81**
Added the symfony 5 support.

### CODE REVIEW

**Comment by chq81** [comment]
@kriswallsmith Hey there. I added an ignore pattern for the missing return types, so now the build passed. If this is fine for you, it would be great if you could merge and tag it :)

**Comment by webhdx** [comment]
ping @Nyholm This pull request as well as a new release is very important.

File: .travis.yml [hunk 1]
```diff
@@ -76,6 +76,9 @@ jobs:
     - name: "Symfony 4.x"
       php: 7.2
       env: DEPENDENCIES="dunglas/symfony-lock:^4"
+    - name: "Symfony 5.x"
+      php: 7.2
+      env: DEPENDENCIES="dunglas/symfony-lock:^5"
```
Comment (Nyholm) [inline .travis.yml]: This does not exist.

Comment (chq81) [reply .travis.yml]: I added the native symfony env variable to lock dependencies.

**Comment by Nyholm** [comment]
Thank you for the reviews and for the PR. 

Symfony 5 support was fixed in #408. Build was fixed in #409 and testing with Symfony 5 added in #410

**Comment by chq81** [comment]
Thanks!

**Comment by chq81** [comment]
Do you plan on tagging the release?

**Comment by Nyholm** [comment]
Yes. 1.1.0 is coming up: https://github.com/kriswallsmith/Buzz/pull/411

**Pull request closed without merging**